test/test_heat_storage.jl

```diff
@@ -96,3 +96,113 @@
     @test real_heatlosses ≈ 1.3333 atol = 0.01
     @test heat_stored ≈ 3.3333 atol = 0.01
 end
+
+@testitem "FixedRateTES" begin
+    using EnergyModelsBase
+    using HiGHS
+    using JuMP
+    using TimeStruct
+
+    const EMH = EnergyModelsHeat
+
+    function generate_data()
+
+        # Define the different resources and their emission intensity in tCO2/MWh
+        power    = ResourceCarrier("Power", 0.0)
+        heat_sur = ResourceCarrier("Heat_surplus", 0.0)
+        heat_use = ResourceCarrier("Heat_usable", 0.0)
+        CO₂      = ResourceEmit("CO₂", 1.0)
+        products = [power, heat_sur, heat_use, CO₂]
+
+        op_duration = 2 # Each operational period has a duration of 2
+        op_number = 4   # There are in total 4 operational periods
+        operational_periods = SimpleTimes(op_number, op_duration)
+
+        op_per_strat = op_duration * op_number
+
+        # Creation of the time structure and global data
+        T = TwoLevel(2, 1, operational_periods; op_per_strat)
+        model = OperationalModel(
+            Dict(CO₂ => FixedProfile(10)),  # Emission cap for CO₂ in t/8h
+            Dict(CO₂ => FixedProfile(0)),   # Emission price for CO₂ in EUR/t
+            CO₂,                            # CO₂ instance
+        )
+
+        nodes = [
+            RefSource(
+                "surplus heat source",      # Node id
+                FixedProfile(0.85),         # Capacity in MW
+                FixedProfile(0),            # Variable OPEX in EUR/MW
+                FixedProfile(0),            # Fixed OPEX in EUR/8h
+                Dict(heat_sur => 1),        # Output from the Node, in this gase, heat_sur
+            ),
+            RefNetworkNode(
+                "HeatPump",
+                FixedProfile(0.5),
+                FixedProfile(0),
+                FixedProfile(0),
+                Dict(heat_sur => 1),
+                Dict(heat_use => 1),
+            ),
+            EMH.FixedRateTES{CyclicStrategic}(
```

Comment:
```suggestion
            FixedRateTES{CyclicStrategic}(
```
EMH should not be require as prefix as we export the type